src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -12,12 +12,17 @@
 
 @if (_trace is { } trace)
 {
-    <div class="trace-detail-layout">
-        <div class="trace-detail-header page-header">
-            <h1 style="display:inline-block">@GetResourceName(trace.FirstSpan.Source): @trace.FirstSpan.Name</h1>
-            <span class="trace-id">@OtlpHelpers.ToShortenedId(trace.TraceId)</span>
-        </div>
-        <FluentToolbar Orientation="Orientation.Horizontal">
+    <AspirePageContentLayout
```

Comment:
The horizontal/vertical button is displayed in mobile view. It doesn't do anything.

![image](https://github.com/dotnet/aspire/assets/303201/7d4801cf-096c-4694-a3ae-4fa67c7d5dc3)

Reply:
removed